### PR TITLE
QRadar v3 add events limit

### PR DESCRIPTION
**Pull request by tomneeman151293**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Added instance parameter for events limit per incident.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Documentation

### CODE REVIEW

**Comment by tomneeman151293** [comment]
Failure is not related to change. it is QRadar v2 test (changes were made in QRadar v3 only)

File: Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.yml [hunk 1]
```diff
@@ -37,6 +37,12 @@ configuration:
   - Fetch Correlation Events Only
   required: true
   type: 15
+- defaultvalue: '20'
+  display: Maximum number of events to pull per incident.
+  name: events_limit
+  required: false
+  type: 0
+  additionalinfo: Relevant for ‘long running instance’.
```
Comment (ShahafBenYakir) [inline Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.yml:45]:
```suggestion
- defaultvalue: '20'
  display: Maximum number of events per incident.
  name: events_limit
  required: false
  type: 0
  additionalinfo: The maximal amount of events to pull per incident.
```
In v2 there was an option to run with long-running or not, in your case it's only long-running

**Review by ShahafBenYakir — APPROVED** [review]
See my comment

**Comment by tomneeman151293** [comment]
@ShahafBenYakir 
The gitlab failed on QRadar v2 test - which is not a part of a change
Also, can I add docs approved?

**Comment by tomneeman151293** [comment]
@ShahafBenYakir Failure is not related to the change (QRadar v2 and QRadar v1 test failures). Can you please force merge?

**Comment by ShahafBenYakir** [comment]
Forcing as change not related to Gitlab crashes

**Pull request merged**